modules/dbfs/filestorage.go

```diff
@@ -0,0 +1,56 @@
+package dbfs
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/CodeCollaborate/Server/modules/config"
+)
+
+var pathsep = strconv.QuoteRune(os.PathSeparator)[1:2]
```

Comment:
I've usually seen this using Filepath.join, instead of building a string literal. I'm not sure if this affects anything, so this is just a comment for now.